DEVELOPMENT.md

```diff
@@ -0,0 +1,178 @@
+# How to work on Buzz About
+
+This is a fairly comprehensive guide to setting up a development environment, aimed at new students. You may already have some of the necessary programs installed&mdash;if so, you can just skip those steps!
+
+This document is draws from prior writeups by [Paul Friederichsen][] and [Prof. Nic McPhee][].
+
+**Table of Contents**
+
+1. Creating a GitHub Account
+2. Installing Visual Studio Code
+3. Installing `git`
+   1. On Windows
+   2. On a Mac
+4. Installing GitKraken
+5. Installing `npm`
+6. Telling `npm` how to talk to GitHub Packages
+   1. Generating a personal access token
+   2. Giving that personal access token to `npm`
+<!-- 7. Installing all of the third-party libraries -->
+
+## Creating a GitHub Account
+
+The code for BuzzAbout&mdash;along with its revision history&mdash;is stored on [GitHub][], a website for hosting software projects. To work on Buzz About, you're going to need to sign up for a GitHub account [here][Sign up for GitHub].
+
+![GitHub's "sign up" page][i:Signing up for GitHub]
+
+You'll want to join the [Minnesota Pollinators][] organization ([Prof. KK Lamberty][] can invite you).
+
+Be sure to sign up for the [student developer pack][Student developer pack], as well!
+
+![The student developer pack's webpage][i:Student developer pack]
+
+## Installing Visual Studio Code
+
+Next, in order to read and write the code, you're going to need a text editor. There are a lot of good options, but the one that we've been using is [Visual Studio Code][], particularly because of its [live share][Live share] feature.
+
+![Visual Studio Code][i:VS Code]
+
+## Installing `git`
+
+`git` is a version-control system: it keeps track of all of the changes made to the project, and lets several people work on the same code in parallel.
+
+### On Windows
+
+If you're on Windows, you can go [here][Install git for Windows] to download `git`.
+
+When the installer asks you to choose the default editor, you may want to select Visual Studio Code. Other than that, the default options should be just fine.
+
+![The git installer program on Windows, asking what the default editor should be, with "Visual Studio Code" selected][i:Windows, installing git]
+
+### On a Mac
+
+If you're on a Mac, you can get `git` as part of the XCode command-line developer tools. Open up Terminal.app and type `git --version`:
```

Comment:
I would suggest just calling it "Terminal" rather than "Terminal.app"

Reply:
Roger that!